projects/ng-components-docs/src/app/examples/components/chip/chip.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+    <div class="col">
+        <sebng-chip [className]="'m-2'" (onClose)="onClose($event)">
```

Comment:
I think you can do this instead
```suggestion
        <sebng-chip className="m-2" (onClose)="onClose($event)">
```